server/mqtt_test.go

```diff
@@ -627,6 +632,18 @@ func TestMQTTParseOptions(t *testing.T) {
 				}
 				return nil
 			}, ""},
+		{"js_api_timeout bad duration", `mqtt: {js_api_timeout: abc}`, nil, "invalid duration"},
```

Comment:
nit: Please move this line higher in the tests list, as part of the "Negative tests" (before the "// Positive tests" comment.

Reply:
oh sorry did not see that :) pushing new change..